src/cpu/flexcpu/inflight_inst.cc

```diff
@@ -52,11 +53,62 @@ InflightInst::InflightInst(ThreadContext* backing_context,
     _pcState(pc_)
 {
     _timingRecord.creationTick = curTick();
+    // Tick's are unsigned, so effectively, -1 is MAX_INT
+    _timingRecord.decodeTick = -1;
+    _timingRecord.issueTick = -1;
+    _timingRecord.beginExecuteTick = -1;
+    _timingRecord.effAddredTick = -1;
+    _timingRecord.beginMemoryTick = -1;
+    _timingRecord.completionTick = -1;
+    _timingRecord.commitTick = -1;
+    _timingRecord.squashTick = -1;
     staticInst(inst_ref);
 }
 
 InflightInst::~InflightInst()
 {
+#if TRACING_ON
```

Comment:
IIRC, no need for this `#if`. The compiler should optimize this away if tracing isn't on.

Reply:
The O3CPU has a similar #if macro. I'll check to see whether it is optimized away.

Reply:
It doesn't *really* matter... I just think it's more readable without it, and I would be shocked if there was a performance difference.